hagglehaul.Server/Models/Trip.cs

```diff
@@ -16,6 +16,9 @@ public class Trip
         [BsonElement("driverEmail")]
         public string DriverEmail { get; set; } = null!;
 
+        [BsonElement("name")]
+        public string Name { get; set; } = null!;
+
```

Comment:
Can we add two new attributes:

`DriverHasBeenRated` bool
`RiderHasBeenRated` bool

These should start out as `false` when a new trip is posted.